### PR TITLE
implemented searchField selector for the involvements

### PR DESCRIPTION
**Pull request by antoniavonto**
Implementing a searchField in advanced people search for searching based on involvements
see: https://github.com/gordon-cs/gordon-360-api/pull/931

### CODE REVIEW

**Comment by antoniavonto** [comment]
Not sure what would be the best icon for involvements
image
image
image

File: package-lock.json [hunk 1]
```diff
@@ -2186,9 +2186,9 @@
       }
     },
     "node_modules/@csstools/media-query-list-parser": {
-      "version": "2.1.0",
-      "resolved": "https://registry.npmjs.org/@csstools/media-query-list-parser/-/media-query-list-parser-2.1.0.tgz",
-      "integrity": "sha512-MXkR+TeaS2q9IkpyO6jVCdtA/bfpABJxIrfkLswThFN8EZZgI2RfAHhm6sDNDuYV25d5+b8Lj1fpTccIcSLPsQ==",
+      "version": "2.1.1",
```
Comment (amos-cha) [inline package-lock.json:2189]: @EjPlatzer not sure if these package versions are supposed to be updated. Can you confirm?

**Review by amos-cha — COMMENTED** [review]
Looks good to me, but the package file is something that needs to be confirmed.

**Comment by antoniavonto** [comment]
@amos-cha

File: package-lock.json
Comment (amos-cha) [inline package-lock.json:1]: can you copy and paste the develop package-lock file and replace the one in here? I don't think there is harm in the updates, but preferably they stayed the same.

Comment (antoniavonto) [reply package-lock.json:1]: done

**Comment by antoniavonto** [comment]
@amos-cha

**Review by amos-cha — APPROVED** [review]

**Pull request merged**